Sources/UIViewExtensions.swift

```diff
@@ -603,35 +603,23 @@ private let UIViewFadeDuration: TimeInterval = 0.35
 
 extension UIView {
     ///EZSE: Fade in with duration, delay and completion block.
-    public func fadeIn(_ duration:TimeInterval?, delay _delay:TimeInterval?, completion: ((Bool) -> Void)? = nil) {
-        UIView.animate(withDuration: duration ?? UIViewFadeDuration, delay: _delay ?? 0.0, options: UIViewAnimationOptions(rawValue: UInt(0)), animations: {
-            self.alpha = 0
-        }) { (success) in
-            if let completion = completion {
-                completion(success)
-            }
-        }
+    public func fadeIn(_ duration:TimeInterval? = UIViewFadeDuration, delay _delay:TimeInterval? = 0.0, completion: ((Bool) -> Void)? = nil) {
```

Comment:
Glups.. eventually I've keep both, I think will be more useful. We can implement view.fadeIn() or view.fadeIn(nil, delay: 1.5, completion: nil) and much more.